lib/robots/dor_repo/gis_assembly/normalize_data.rb

```diff
@@ -263,11 +263,12 @@ def convert_8bit_to_rgb
           end
 
           def eight_bit?
-            cmd = "#{Settings.gdal_path}gdalinfo -norat -noct '#{output_filepath}'"
-            infotxt = IO.popen(cmd, &:readlines)
-            infotxt.each do |line|
-              # Band 1 Block=4063x2 Type=Byte, ColorInterp=Palette
-              return true if line.match?(/^Band (.+) Block=(.+) Type=Byte, ColorInterp=Palette\s*$/)
+            cmd = "#{Settings.gdal_path}gdalinfo -json -norat -noct '#{output_filepath}'"
+            IO.popen(cmd).read.tap do |gdalinfo_json_str|
+              gdalinfo_json = JSON.parse(gdalinfo_json_str)
+              bands = gdalinfo_json['bands']
+              # { "bands":[{ "band": 1, "block": [10503, 3], "type": "Byte", "colorInterpretation": "Palette" }] } # plus many other keys at each level
+              return true if bands.any? { |band| band.key?('block') && band['type'] == 'Byte' && band['colorInterpretation'] == 'Palette' }
```

Comment:
`normalize_data_spec.rb` was already shelling out to `gdalinfo` before this PR, so i didn't have to shore up the tests for this.